fs_overlay/var/lib/nginx-conf/nginx.conf.erb

```diff
@@ -132,4 +132,12 @@ http {
             ssl_reject_handshake on;
         }
     <% end %>
+
+    <% unless ENV['NGINX_STATS'] == 'off' %>
```

Comment:
I'm deadly sure that it should not be enabled by default. Especially if you are turning access logging off.